File: video/rooms/participants/retrieve-participant-list-disconnected/retrieve-participants-disconnected.java
import com.twilio.Twilio;
import com.twilio.base.ResourceSet;
import com.twilio.rest.video.v1.room.Participant;

public class ParticipantsExample3 {

	public static final String API_KEY_SID = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
	public static final String API_KEY_SECRET = "your_api_key_secret";

	public static void main(String args[]) {
		Twilio.init(API_KEY_SID, API_KEY_SECRET);

		ResourceSet<Participant> participants = Participant.reader("RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
				.setStatus(Participant.Status.DISCONNECTED)
				.read();

		for (Participant participant : participants) {
			System.out.println(participant.getIdentity());
		}
	}
}

File: video/rooms/participants/retrieve-participant-list-disconnected/retrieve-participants-disconnected.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require __DIR__ . '/twilio-php-master/Twilio/autoload.php';  // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$apiKeySid = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$apiKeySecret = "your_api_key_secret";
$client = new Client($apiKeySid, $apiKeySecret);

$participants = $client->video->rooms("RMXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")->participants->read(array("status" => "disconnected"));

foreach ($participants as $participant) {
    echo $participant->duration;
}
